File: Cpp/9_Palindrome_Number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x < 0) return false;
        
        long reverted = 0;
        int n = x;
        while(n != 0){
            reverted = reverted * 10 + (n % 10);
            n/=10;
        }
        
        return reverted == x;
    }
};

File: Kotlin/9_Palindrome_Number.kt
class Solution {
    fun isPalindrome(x: Int): Boolean {
        if(x < 0) return false
        
        var reverted = 0
        var n = x
        while(n != 0){
            reverted = reverted * 10 + (n % 10)
            n /= 10
        }
        
        return reverted == x
    }
}